scripts/sct_analyze_texture.py

```diff
@@ -22,68 +22,81 @@
 import sct_utils as sct
 import spinalcordtoolbox.image as msct_image
 from spinalcordtoolbox.image import Image
-from msct_parser import Parser
+import argparse
```

Comment:
argparse is in the Python standard lib -> import it in the first block of imports. Order is stdlib / 3rd party / internal.

Reply:
Ok sounds good! Changed in commit [54d0476](https://github.com/neuropoly/spinalcordtoolbox/commit/54d04763befd566da8b96f2ee6e80071a4de08d9)